BoosterKit/Library/APIManager.swift

```diff
@@ -46,4 +59,37 @@ class APIManager {
             }
         }
     }
+
+    public class func authenticateUser (
+        email: String,
+        password: String,
+        passwordConfirmation: String,
+        completionClosure: @escaping () -> Void) {
+        let parameters = [
+            "email": email,
+            "password": password,
+            "password_confirmation": passwordConfirmation
+        ]
+
+        Alamofire.request(
+            "\(Constants.apiURL)/auth/sign_in",
+            method: .post,
+            parameters: parameters,
+            encoding: JSONEncoding.default
+            ).validate()
+            .responseJSON { response in
+                logger.debug("Sign In: \(response.result.isSuccess)")
+                switch response.result {
+                case .success:
+                    if let headers = response.response?.allHeaderFields as? [String: String] {
+                        keychain["Access-Token"] = headers["Access-Token"]
+                        keychain["Client"] = headers["Client"]
+                        keychain["Uid"] = headers["Uid"]
+                    }
+                    completionClosure()
+                case .failure(let error):
+                    logger.error(error)
```

Comment:
I personally like to pass the error back to the completion block so anyone using the API can handle failures as needed. The logging is good for debugging though.